receiver/oracledbreceiver/documentation.md

```diff
@@ -0,0 +1,51 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# oracledbreceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| **oracledb.session.cpu_usage** | CPU usage (%) | 1 | Gauge(Double) | <ul> </ul> |
```

Comment:
The description is a it unclear. Is this percentage? If unit is `1` I believe it must be fraction of a core. Also we usually use term `utilization` for CPU in other receivers

Reply:
Took it out - now using a cumulative metric in seconds